packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -1771,12 +1790,17 @@ class _SelectableRegionContainerDelegate extends MultiSelectableSelectionContain
 /// [Selectable]s that currently contain the selection edges.
 abstract class MultiSelectableSelectionContainerDelegate extends SelectionContainerDelegate with ChangeNotifier {
   /// Creates an instance of [MultiSelectableSelectionContainerDelegate].
-  MultiSelectableSelectionContainerDelegate() {
+  MultiSelectableSelectionContainerDelegate({
+     required this.copyInterceptor
+  }) {
     if (kFlutterMemoryAllocationsEnabled) {
       ChangeNotifier.maybeDispatchObjectCreation(this);
     }
   }
 
+  /// The [CopyInterceptor] to use for handling copy requests.
```

Comment:
This comment is kind of redundant, maybe instead something like: "Specifies how [Selectables] are serialized into a string when copied."